doc/cephfs: improve cache-configuration.rst (#59168)
ceph/ceph: doc/cephfs/cache-configuration.rst

mds_min_caps_per_client(100)``).

Under certain circumstances, many "recall caps" requests can be sent so quickly

> that the "mon warning limit" is exceeded, and the "clients failing to respond
> to cache pressure" message can be triggered. If the client does not release the

Reviewer: ```suggestion
that the health warning is generated: "clients failing to respond
to cache pressure". If the client does not release the
```

Author: I've committed this change. Thanks, @batrick.

Example Scenario
~~~~~~~~~~~~~~~~

> Here is an example. A client is having 20,000 caps cached. At some moment the

Reviewer: Given that different locales use "," or ".", would it not be more understandable to use an SI prefix?

Author: What do you think about "20K"?

Reviewer: The prefixes are generally case-sensitive so I think "k" is still appropriate: https://en.wikipedia.org/wiki/Metric_prefix
